pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1006,6 +999,7 @@ var _ = Describe("ProjectService", func() {
 				})
 
 				It("warns the user and defaults restart policy to `Always`", func() {
+					Expect(projectService.K8SConfig.Workload.RestartPolicy).To(Equal(policy))
```

Comment:
Are we expecting workload RestartPolicy defined in x-k8s as "unless-stopped" here i.e. compose notation? or one of the values we defined for it in label based configuration? See: https://github.com/appvia/kev/blob/master/docs/reference/config-params.md#kevworkloadrestart-policy

Reply:
k8s should hold whatever is inferred from compose and then merged with extensions content. So both cases can happen.

Reply:
OK, I guess we just need to ensure that what we derive from docker compose is translated into values we want the end user to see/tweak. Currently we let the user to adjust configuration based on the config params we defined - see link above. 

And these are the only values we should be populating in the `projectService.K8SConfig.Workload.RestartPolicy`.  

The spec above expects that the value of `projectService.K8SConfig.Workload.RestartPolicy` is `unless-stopped` which I don't think is correct as it should be one of allowed values: `Always`, `OnFailure`, `Never`.

Reply:
so if I understand correctly, you're proposing we move [this](https://github.com/appvia/kev/blob/8621035902e12a1d72078badb617cc5a784d3279/pkg/kev/converter/kubernetes/project_service.go#L457) to the config.K8SCfgFromCompose?